nncf/quantization/algorithms/min_max/onnx_backend.py

```diff
@@ -268,3 +268,7 @@ def get_weight_name(nncf_graph: NNCFGraph, target_point: ONNXTargetPoint) -> str
     def should_quantize_weight(weight_name: str, quantized_weight_names: Set[str]) -> bool:
         # If the nodes share one weight tensor, we should have only one quantizer on that
         return weight_name not in quantized_weight_names
+
+    @staticmethod
+    def is_matmul_with_constant(node: NNCFNode, nncf_graph: NNCFGraph) -> bool:
+        return node.layer_attributes.has_weight()
```

Comment:
Add check to matmul metatype in this function for all backend

Reply:
Done